src/main/java/seedu/address/storage/rooms/RoomRecordsStorage.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.storage.rooms;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.model.ReadOnlyList;
+import seedu.address.model.room.Room;
+
+//@@author itssodium
+public interface RoomRecordsStorage {
+
+    /**
+     * Returns the file path of the data file.
+     */
+    Path getRoomsRecordsFilePath();
+
+    /**
+     * Returns CovigentApp data as a {@code ReadOnlyList<Patient>}.
```

Comment:
i think this should be `{@code ReadOnlyList<Room>}` instead of `{@code ReadOnlyList<Patient>}`?